Lib/fontParts/base/normalizers.py

```diff
@@ -306,7 +310,7 @@ def normalizeGlyphUnicode(value):
     * **value** must be in a unicode range.
     * Returned value will be an ``int``.
     """
-    if not isinstance(value, (int, basestring)):
+    if not isinstance(value, (int, basestring)) or type(value) is bool:
```

Comment:
`type(value) is bool`: why not`isinstance(value, bool)` (codacy complained about this)

Reply:
I'll change it

Reply:
done. I also took the opportunity to do formatting fix-ups on that file